drivers/storage/portworx/portworx_test.go

```diff
@@ -254,9 +254,9 @@ func TestSetDefaultsOnStorageCluster(t *testing.T) {
 	driver.SetDefaultsOnStorageCluster(cluster)
 
 	// Use default image from release manifest when spec.image is not set
-	require.Equal(t, defaultPortworxImage+":3.0.0", cluster.Spec.Image)
-	require.Equal(t, "3.0.0", cluster.Spec.Version)
-	require.Equal(t, "3.0.0", cluster.Status.Version)
+	require.Equal(t, defaultPortworxImage+":2.10.0", cluster.Spec.Image)
```

Comment:
Can we add some tests in this function - https://github.com/libopenstorage/operator/blob/894e0117f6f130e9a623b77c4d9c55369a8724da/drivers/storage/portworx/portworx_test.go#L7264?